astraceroute.c

```diff
@@ -303,7 +303,7 @@ static void __noreturn help(void)
 	     " -6|--ipv6               Use IPv6-only requests\n"
 	     " -n|--numeric            Do not do reverse DNS lookup for hops (default)\n"
 	     " -u|--update             Update GeoIP databases\n"
-	     " -L|--latitude           Show do_geo_lookup and longitude\n"
+	     " -L|--latitude           Show latitude and longitude\n"
```

Comment:
Please squash this with the commit (in your branch) that introduced the typo.

Reply:
Done.